src/components/05_pages/NodeForm/Widgets.js

```diff
@@ -0,0 +1,7 @@
+import StringTextfield from '../../02_atoms/Widgets/StringTextfield';
+
+const widgets = {
+  string_textfield: StringTextfield,
```

Comment:
For now this is hard coded but I could imagine that this will be part of the build step and generated at some point.